examples/annotate_segmentation_with_text.py

```diff
@@ -3,7 +3,9 @@
 ===============================
 
 Perform a segmentation and annotate the results with
-bounding boxes and text
+bounding boxes and text.
+This example is fully explained in the following tutorial:
+https://napari.org/stable/tutorials/segmentation/annotate_segmentation.html
```

Comment:
We really should have an intersphinx link for this, but pending a PR to the docs repo this looks good!

Reply:
Hmm. that would make the gallery nicer right?
But the intersphinx wouldn't work for someone looking at the example in the repo?

Reply:
Yeah, you are right. In this case It may be better to use the full URL indeed.